PIPELINES, Win32, same failure twice.
======================================================================
FAIL: test_incremental_editing (idlelib.idle_test.test_colorizer.ColorDelegatorTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "D:\a\1\s\lib\idlelib\idle_test\tkinter_testing_utils.py", line 54, in new_test_method
    raise exception
  File "D:\a\1\s\lib\idlelib\idle_test\tkinter_testing_utils.py", line 38, in after_callback
    next(test_generator)
  File "D:\a\1\s\lib\idlelib\idle_test\test_colorizer.py", line 597, in test_incremental_editing
    eq(text.tag_nextrange('KEYWORD', '1.0'), ())
AssertionError: Tuples differ: ('1.0', '1.1') != ()

First tuple contains 2 additional elements.
First extra element 0:
'1.0'

- ('1.0', '1.1')
+ ()

Why only Pipelines and not others? Why only Win32 and not Win64?  I am tempted to ignore, but I will trigger a rerun.

--- a/Lib/idlelib/autocomplete_w.py
+++ b/Lib/idlelib/autocomplete_w.py
@@ -243,10 +243,8 @@ class AutoCompleteWindow:
         if not self.is_active():
             return
 
-        # Due to the <Configure> event sometimes being triggered after some
-        # delay, the completion list window may no longer exist when the
-        # handler is called.  Therefore it is necessary to catch potential
-        # TclError exceptions in the handler's code.  See: bpo-41611.
+        # Since the <Configure> event may occur after the completion window is gone,
+        # catch potential TclError exceptions when accessing acw.  See: bpo-41611.
         try:
             # Position the completion list window
             text = self.widget

